packages/protocol/contracts/common/Blockable.sol

```diff
@@ -0,0 +1,41 @@
+pragma solidity >=0.5.13 <0.9.0;
+
+import "./interfaces/IBlockable.sol";
+import "./interfaces/IBlocker.sol";
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+
+contract Blockable is IBlockable, Ownable {
```

Comment:
NIT: Might also be worth adding NATSPEC so we dont have to go back and do it later